GVRf/Framework/jni/objects/textures/compressed_texture_jni.cpp

```diff
@@ -30,7 +30,7 @@ Java_org_gearvrf_asynchronous_NativeCompressedTexture_normalConstructor(JNIEnv *
 
 JNIEXPORT jlong JNICALL
 Java_org_gearvrf_asynchronous_NativeCompressedTexture_mipmappedConstructor(JNIEnv * env,
-        jobject obj, jint target);
+        jobject obj, jint target, jintArray jtexture_parameters);
```

Comment:
By design, we didn't force to pass texture_parameters in the texture constructors (Texture, CompressedTexture, FloatTexture, etc). But the wrapping parameters can be passed using the Texture::updateTextureParameters function. They include wrap_s_type and wrap_t_type already. So please try to use the existing API.